locals.js

```diff
@@ -16671,6 +16852,38 @@ I18N.zh["sponsors"] = { // 赞助界面
                 "GitHub Sponsors eligible accounts": "符合 GitHub 赞助者资格的账户",
                     "This account has not applied to join GitHub Sponsors.": "此账户尚未申请加入 GitHub 赞助者。",
                     "Join the waitlist": "加入等候名单",
+
+        // 赞助者资料页 https://github.com/sponsors/<user-name>
```

Comment:
```suggestion
        // GitHub 赞助者个人资料 https://github.com/sponsors/<user-name>
        // GitHub 赞助者组织资料 https://github.com/sponsors/<org-name>
        // ...?preview=true
            "You are previewing your GitHub Sponsors profile.": "您正在预览您的 GitHub 赞助者个人资料。",
            "Edit your profile.": "编辑您的个人资料。",
            // 组织
            // [/You are previewing ([^ ]+)’s GitHub Sponsors profile./, "您正在预览 $1 组织的 GitHub 赞助者个人资料。"],
            // [/Edit ([^ ]+)’s profile./, "编辑 $1 组织的个人资料。"],
```
![image](https://github.com/user-attachments/assets/d2d7e1d4-6c21-4173-8f2c-7870a33403de)

Reply:
另外，在
```
        [/([^ ]+) does not directly depend on any repositories whose maintainers can be sponsored./, "$1 不直接依赖于任何可以赞助其维护人员的仓库。"],
```
之上，添加正则
```
        [/You are previewing ([^ ]+)’s GitHub Sponsors profile./, "您正在预览 $1 组织的 GitHub 赞助者个人资料。"], // sponsors/<org-name>?preview=true
        [/Edit ([^ ]+)’s profile./, "编辑 $1 组织的个人资料。"], // sponsors/<org-name>?preview=true
```